iDiff/utils/package_diff_utils.go

```diff
@@ -0,0 +1,71 @@
+package utils
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"reflect"
+)
+
+// PackageDiff stores the difference information between two images.
```

Comment:
just a little thing, but I feel like the image fields can be renamed to be generalized more in case someone would want to use/build off of this outside of the realm of docker images... similarly, the BuildLayerTargets method below is very specific to docker images as well, so perhaps we can move it elsewhere when we talk about refactoring